daemon/src/model/cfd.rs

```diff
@@ -731,7 +728,8 @@ impl Cfd {
                 self.quantity,
                 self.leverage,
                 self.refund_timelock_in_blocks(),
-                tx_fee_rate,
+                tx_fee_rate, // TODO: Where should I get the fee rate from?
```

Comment:
Maybe best is if the maker can provide this when accepting rollover?

Reply:
I think he already does, this is a rebase issue where I re-introduced a comment that should be gone :)